README.md

```diff
@@ -58,6 +58,22 @@ module.exports = function (environment) {
 };
 ```
 
+#### `page-title` Service
+
+If you want to be notified when the page title has been updated, you can extend and override the `page-title` service and provide your own `titleDidUpdate` hook. The `titleDidUpdate` hook receives the new title as its sole argument.
+
+```javascript
+// app/services/page-title.js
+
+import EmberPageTitleService from 'ember-page-title/services/page-title';
+
+class PageTitleService extends EmberPageTitleService {
```

Comment:
Let's add also 'export default' so copy pasting this example would work out of the box.

Reply:
Good catch

Reply:
Yikes, how could I forget 🤦.